okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -271,8 +271,7 @@ SSLSocketFactory createSocketFactory() {
       case TLS:
         try {
           if (sslSocketFactory == null) {
-            SSLContext sslContext = SSLContext.getInstance("TLS", Platform.get().getProvider());
-            sslContext.init(null, null, null);
+            SSLContext sslContext = SSLContext.getInstance("Default", Platform.get().getProvider());
```

Comment:
Not. More about minimising the delta from original behavior

-louis (from phone)

On Jul 15, 2016 1:36 PM, "Eric Anderson" <notifications@github.com> wrote:

> In okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java
> <https://github.com/grpc/grpc-java/pull/2049#discussion_r71037048>:
>
> > @@ -271,8 +271,7 @@ SSLSocketFactory createSocketFactory() {
> >        case TLS:
> >          try {
> >            if (sslSocketFactory == null) {
> > -            SSLContext sslContext = SSLContext.getInstance("TLS", Platform.get().getProvider());
> > -            sslContext.init(null, null, null);
> > +            SSLContext sslContext = SSLContext.getInstance("Default", Platform.get().getProvider());
>
> Interesting... Is that a fix for an issue?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/grpc/grpc-java/pull/2049/files/a8fee8cd03e9992634c729519e4e7a2c6b68c1ba#r71037048>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIoKPEZry4cdIQR-OIPYjiJFPkhDDKrRks5qV-81gaJpZM4JNsgc>
> .
>